lib/sensors/i2c_sensors.hpp

```diff
@@ -16,7 +16,6 @@ class II2cMuxSensor {
   /*
    * This function carries out the initilization steps for a particular sensor.
    */
-  virtual core::Result configure()        = 0;
   virtual std::optional<T> read()         = 0;
```

Comment:
Do all these sensors not use a `create` function too?

Reply:
yes, so they would be pre-configured and the mux itself would just accept a pointer to them